_src/pattern-library/sass/utilities/_variables.scss

```diff
@@ -22,32 +22,26 @@ $em-base: 16; // deliberately sets bourbon-based em-base (http://bourbon.io/docs
 // #GRID
 // ----------------------------
 // grid - baseline
-$baseline:               20 !default;
-$baseline-vertical:      ($baseline*2) !default;
-$baseline-horizontal:    $baseline !default;
-
-$susy: (
-    columns: 12,
-    container: rem(1170),
-    global-box-sizing: border-box,
-    gutters: .5,
-    math: fluid,
-    output: float,
-    use-custom: (
-        breakpoint: true,
-        background-image: true,
-        background-options: true,
-        box-sizing: false,
-        clearfix: true,
-        rem: true,
-    )
-);
+$baseline:                   20 !default;
+$baseline-vertical:          ($baseline*2) !default;
+$baseline-horizontal:        $baseline !default;
+
+// grid - dimensions
+$grid-max-width:             rem(1170) !default;
+$grid-column-count:          12 !default;
+$grid-gutter-width:          (1/3) !default;  // set to fraction to be proportional to fluid column widths (see - http://susydocs.oddbird.net/en/latest/settings/#gutters)
+$grid-container-position:    center !default;
+$grid-gutter-position:       after !default;
+$grid-manual-gutter-position: inside !default;
+$grid-column-width:          false !default;  // set to false for fluid columns by default (see - http://susydocs.oddbird.net/en/latest/settings/#column-width)
```

Comment:
NOTE: We're letting Susy calculate %-based column width from our ``$grid-max-width``, ``$grid-column-count``, and ``$grid-gutter-width``. Adding this in case folks want to set a static grid with widths.